boards/esp32c3/kitten_grapebit_c3.board.json

```diff
@@ -0,0 +1,61 @@
+{
+    "$schema": "../esp32deviceconfig.schema.json",
+    "devName": "Kittenbot Grape:Bit ESP32-C3 DeviceScript",
+    "productId": "0x3635315b",
+    "$description": "Stemcraft C3",
+    "jacdac": {
+        "$connector": "Jacdac",
+        "pin": 5
+    },
+    "led": {
+        "pin": 8,
+        "activeHigh": true
+    },
+    "i2c": {
+        "$connector": "Qwiic",
+        "pinSCL": 7,
+        "pinSDA": 6
+    },
+    "pins": {
+        "P1": 2
+    },
+    "log": {
+        "pinTX": 0,
```

Comment:
Oh, and logging on C3 always goes to pin 21 (if it goes anywhere) - this is where the bootloader prints.